### PR TITLE
docs: adds powerbi dedicated docs

### PR DESCRIPTION
**Pull request by SurtzSean**
https://github.com/tomkerkhove/promitor/pull/2368
https://github.com/tomkerkhove/promitor/issues/339

### CODE REVIEW

File: docs/scraping/providers/powerbi-embedded.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - Scraper
+  - Resource Discovery
+  - PowerBi
+---
+
+# Azure PowerBi Embedded
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.9-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```
Comment (tomkerkhove) [inline docs/scraping/providers/powerbi-embedded.md:10]:
```suggestion
![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.11-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```

File: docs/scraping/providers/powerbi-embedded.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+You can declare to scrape a PowerBi Embedded capacity via the `PowerBiEmbedded` resource-type.
```
Comment (tomkerkhove) [inline docs/scraping/providers/powerbi-embedded.md:12]: Is `PowerBiEmbedded` the correct fit? It looks like it's only for dedicated power BI?

Also, is Power BI Embedded not deprecated? I might be remembering this wrong or confuse with another Azure offering of Power BI

Comment (SurtzSean) [reply docs/scraping/providers/powerbi-embedded.md:12]: @tomkerkhove any preference on the naming?

Comment (SurtzSean) [reply docs/scraping/providers/powerbi-embedded.md]: sounds good, I updated the naming here, I'll make an update to the main PR later tonight

**Review by tomkerkhove — APPROVED** [review]

File: docs/scraping/providers/powerbi-dedicated.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+All supported metrics are documented in the official [Azure Monitor documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-powerbidedicated-capacities-metrics).
+
+## Example
+
+Here is an example configuration:
+
+```yaml
+  - name: azure_powerbi_dedicated_cpu_usage
+    description: percentage of cpu used by powerbi dedicated
+    resourceType: PowerBiEmbedded
```
Comment (SurtzSean) [inline docs/scraping/providers/powerbi-dedicated.md:28]: I think resource type must still be powerbi embedded, at least from my testing.

Comment (tomkerkhove) [reply docs/scraping/providers/powerbi-dedicated.md:28]: As discussed [here](https://github.com/promitor/docs/pull/54#discussion_r1303979971), PowerBiDedicated would be a better name to be more consistent

File: docs/scraping/providers/powerbi-dedicated.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  - Resource Discovery
+  - PowerBI
+  - PowerBI Embedded
+---
+
+# Azure PowerBI Dedicated
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.11-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
+
+You can declare to scrape a PowerBi Dedicated capacity via the `PowerBiDedicated` resource-type.
```
Comment (tomkerkhove) [inline docs/scraping/providers/powerbi-dedicated.md:13]:
```suggestion
You can declare to scrape a Azure PowerBI Dedicated capacity via the `PowerBiDedicated` resource type.
```

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**